detailed_cost_report.py

```
@@ -0,0 +1,216 @@
+    total_videos = len(videos_with_duration)
+    total_seconds = sum(v['duration_seconds'] for v in videos_with_duration)
+    total_minutes = total_seconds / 60
+    total_hours = total_minutes / 60
+    avg_duration = total_minutes / total_videos if total_videos > 0 else 0
+    
+    # Current cost analysis
+    current_cost = 9.059
+    cost_per_minute = current_cost / total_minutes if total_minutes > 0 else 0
+    cost_per_video = current_cost / total_videos if total_videos > 0 else 0
+    
+    print(f"\n📋 RESUMEN EJECUTIVO")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)